README.md

```diff
@@ -10,7 +10,12 @@ Granite is the SpongeAPI implementation for Vanilla Minecraft.
 * [Gradle] 2.3+
 
 ## Cloning
-To clone Granite, simply run `git clone git@github.com:SpongePowered/Granite.git`
+The following steps will ensure your project is cloned properly.
+
+1. `git clone git@github.com:SpongePowered/Granite.git`
```

Comment:
I get the SpongePowered logic: https://github.com/SpongePowered/Sponge#clone
We can add a or for this like:
1. `git clone git@github.com:SpongePowered/Granite.git` or `git clone https://github.com/SpongePowered/Granite.git`